service/src/main/java/uk/gov/hmcts/reform/fpl/service/orders/generator/C43aSpecialGuardianshipOrderDocumentParameterGenerator.java

```diff
@@ -81,7 +82,20 @@ private String getSpecialGuardianAppointeeMessage(CaseData caseData, int numOfCh
         String childOrChildren = (numOfChildren == 1 ? "child" : "children");
         String applicant = appointedGuardianFormatter.getGuardiansNamesForDocument(caseData);
 
-        return format("The Court orders that %s appointed as special guardian for the %s.", applicant, childOrChildren);
+        ManageOrdersEventData eventData = caseData.getManageOrdersEventData();
+
+        StringBuilder sb = new StringBuilder();
+        if (isNotEmpty(applicant)) {
+            sb.append(format("The Court orders that %s appointed as special guardian for the %s.", applicant,
+                    childOrChildren));
+        }
+        if (isNotEmpty(eventData.getAppointedGuardianDetails())) {
```

Comment:
Just to double check, would we expect there to be a situation where the user uses both the free text field and selects the guardians from the drop box?

Is it needed to add additional guardians to the ones list above?